### DISCUSSION TITLE
Is it possible to work with the Amazon Shipping API in test mode without registration ?

### DISCUSSION THREAD

**Discussion by sergei-kh** [amzn/selling-partner-api-models#2820] [General] [answered]
Hello, for employment, I am doing a test task for working with the Amazon Shipping API, I need to send a test order and get a tracking number.
Do I need to register with AWS and accept the IAM policy in order to use the API? Unfortunately, I don't have a working bank card yet and can't register with AWS. I wanted to clarify, maybe there is some other way that allows you to work with the Amazon Shipping API in test mode without registration ?

**Comment by sungolivia** [comment, marked as answer]
hey @sergei-kh  unfortunately you would need AWS account which you would need working credit card even for the AWS free tier. 😢

**Reply by sergei-kh** [reply]
understand, thanks
